src/main/java/com/servicewizard/locator/JerseyResourceLocator.java

```diff
@@ -64,6 +67,7 @@ public JerseyResourceLocator(String packageName) {
 	 * Creates a Service object based on the annotations present on the given class
 	 */
 	private Service buildService(Class<?> serviceClass) {
+		Log.info("Checking resource: " + serviceClass.getName());
```

Comment:
I think "checking" implies that behavior is conditional. But there will be a side-effect regardless of whether there are methods found inside, so I think "discovered" could be preferable here.